src/main/java/dan200/computercraft/core/apis/TermAPI.java

```diff
@@ -101,6 +111,59 @@ public static int parseColour( Object[] args, boolean _enableColours ) throws Lu
         };
     }
 
+    public static void setColour( Terminal terminal, int colour, float r, float g, float b )
+    {
+        if( terminal.getPalette() != null )
+        {
+            terminal.getPalette().setColour( colour, r, g, b );
+            terminal.setChanged();
+        }
+    }
+
+    public static void setColour( Terminal terminal, HashMap<Object, Object> colours) throws LuaException
```

Comment:
Sure, but `window` isn't the only thing in the universe that uses `setPaletteColour`. But alright, I'll get rid of it, we can always add it back if needed, after all.